flipperdevices/flipperzero-firmware
# NFC: Fixed writing gen1a magic tags with invalid BCC

# What's new

- Fixed writing gen1a magic tags with invalid BCC

Note: now furi_hal_nfc_detect is used only for checking non magic tags, magic tag writing happens without anti-collision (like 'Check Magic tag')
# Verification 

- Try writing gen1a tag with invalid BCC (not readable via Flipper NFC app, but visible in 'Check Magic tag'). Tag should be writable.
- Try writing gen1a tag with valid BCC. Tag also should be writable.
- Try writing non gen1a tag. App should show that the card isn't magic.

# Checklist (For Reviewer)
